main.go

```diff
@@ -2,11 +2,11 @@ package main
 
 import "fmt"
 
-func writeMsg(a, b int) {
-	fmt.Print(a + b)
+func writeMsg(a, b, c, d int) {
```

Comment:
The argument list to this function is getting quite long. It should instead take a dataset.